cloudslang-cli/src/main/java/io/cloudslang/lang/cli/services/SyncTriggerEventListener.java

```diff
@@ -107,7 +110,12 @@ else if(data.containsKey(LanguageEventData.OUTPUTS)
                         && data.containsKey(LanguageEventData.PATH)
                         && data.get(LanguageEventData.PATH).equals(EXEC_START_PATH)) {
                     Map<String, Serializable> outputs = extractNotEmptyOutputs(data);
-                    printWithColor(Ansi.Color.WHITE, "\nFlow outputs:");
+                    if (data.containsKey(ScoreLangConstants.EXECUTABLE_TYPE) &&
+                            ExecutableType.OPERATION.equals(data.get(ScoreLangConstants.EXECUTABLE_TYPE))) {
```

Comment:
you can remove `data.containsKey(ScoreLangConstants.EXECUTABLE_TYPE) &&` from the condition